app/models/user.rb

```diff
@@ -20,6 +20,13 @@ class User < ApplicationRecord
   scope :with_email, -> { where.not email: nil }
 
   scope :name_order, -> { order :last_name, :first_name }
+  scope :in_divisions, (lambda do |divisions|
+    joins(:divisions_users).where(divisions_users: { division_id: divisions.pluck(:id) })
+  end)
+
+  def division
+    divisions.first
+  end
```

Comment:
This was a "path of least resistance" change. Drivers will only belong to one division.

Reply:
This is only used for `delegate :division, to: :driver` in `Incident`.

Reply:
> Drivers will only belong to one division

Is that something you want to enforce?

Reply:
Hmm. Yeah, I guess I should.

Reply:
Not a blocking concern for me but it could be a simple model validation if you wanted to add it.